### DISCUSSION TITLE
Is it too slow or Am I doing something wrong?

### DISCUSSION THREAD

**Discussion by rstulug** [isce-framework/isce2#906] [General]
I used 30 SLC images and stackSentinel.py created 110 interfereogram when choose -c 4. The parameter --num_proc is chosen 56 (I have 56 core) to speed up the process but It has not finished for 3 days. Everthing looks correct, there is no error messages and the program creates necessery files one by one. Furthermore when calling run.py, I also specified -p 56. Below you can see the code that I used to create run files.  If there is an error that I made, how can fix it or if it the same for everyone, Is there a way to decrease the computation time?

stackSentinel.py ./SLC/ -d ./DEM/demLat_N38_N39_Lon_E030_E032.dem.wgs84 -a ./AuxDir/ -o ./Orbits/ -b "38 39 30.25 32" -C geometry -c 4 --num_proc 56 -W interferogram

**Comment by Sugeradi985** [comment]
Some tasks, such as filtering and unwrapping, require a significant amount of memory. 
Based on my experience, your study area requires at least 10GB of memory for each interferogram when filtering, so make sure you have enough memory space. 
Additionally, you should check your task manager to see if any tasks have been killed.

**Reply by rstulug** [reply]
I have enough ram for the task and controlled system monitor but nothing is changed. Maybe it is related to the my processor but I am not sure about that. By the way, thanks for reply.
